src/export/FSHExporter.ts

```diff
@@ -150,32 +154,46 @@ export class FSHExporter {
     return new Map().set('resources.fsh', results);
   }
 
-  private groupAsFilePerDefinition(): Map<string, Exportable[]> {
+  private groupAsFilePerDefinition(byGroup = false): Map<string, Exportable[]> {
```

Comment:
If you do make this `file-per-definition`, added bonus is this function gets simpler.